lib/elixir/test/elixir/list_test.exs

```diff
@@ -257,4 +257,39 @@ defmodule ListTest do
       List.improper?(%{})
     end
   end
+
+  describe "ascii_printable?/1" do
+    test "proper lists without limit" do
+      assert List.ascii_printable?([])
+      assert List.ascii_printable?('abc')
+      refute(List.ascii_printable?('abc' ++ [0]))
+      assert List.ascii_printable?('abc\a\b\e\f\n\r\t\vdef')
+      refute List.ascii_printable?('mañana')
+    end
+
+    test "proper lists with limit" do
+      assert List.ascii_printable?([], 100)
+      assert List.ascii_printable?('abc' ++ [0], 2)
+    end
+
+    test "improper lists" do
+      refute List.ascii_printable?('abc' ++ ?d)
+      assert List.ascii_printable?('abc' ++ ?d, 3)
+      assert List.ascii_printable?([1, 2 | 3], 0)
+    end
+
+    test "function clause errors" do
```

Comment:
We generally don't test function clause errors.

Reply:
should i spread those tests around the other test blocks? or remove the tests